commands/team.go

```diff
@@ -354,16 +356,19 @@ func modifyTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 
 func restoreTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	teams := getTeamsFromTeamArgs(c, args)
+	var result *multierror.Error
 	for i, team := range teams {
 		if team == nil {
+			result = multierror.Append(result, fmt.Errorf("unable to find team '%s'", args[i]))
 			printer.PrintError("Unable to find team '" + args[i] + "'")
 			continue
 		}
 		if rteam, _, err := c.RestoreTeam(team.Id); err != nil {
+			result = multierror.Append(result, errors.New("Unable to restore team '"+team.Name+"' error: "+err.Error()))
```

Comment:
Please use `fmt.Errorf` here as well, that way you can use string interpolation like in line 362.

Reply:
Done.